worker/remoterelations/doc.go

```diff
@@ -0,0 +1,9 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Package remoterelations contains workers which manage the operation of cross model relations.
```

Comment:
Can we enumerate the multiple workers and describe what each one does?

Reply:
Also, maybe better to say: defines* workers that...

Reply:
Workers don't have names. We can attempt to describe them by the struct the implements them.